src/App.js

```diff
@@ -1,12 +1,35 @@
 import React from 'react';
-import Calculator from './components/Calculator';
+import { Route, Switch } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Home from './pages/Home';
+import Cal from './pages/Cal';
+import Quote from './pages/Quote';
 
 function App() {
   return (
-    <div>
-      <h1>Math Magicians</h1>
-      <Calculator />
-    </div>
+    <>
```

Comment:
- [ ] The application throws console errors. (Seems like it is related to an input field) Please be sure you have fixed all console errors. 

![Screenshot from 2021-12-03 02-07-41](https://user-images.githubusercontent.com/1642014/144517373-e059dfad-2054-439e-9fb3-e4ab9492e702.png)